configure.ac

```diff
@@ -2334,7 +2334,7 @@ AS_CASE([$ac_sys_system],
 
     dnl Include file system support
     AS_VAR_APPEND([LINKFORSHARED], [" -sFORCE_FILESYSTEM -lidbfs.js -lnodefs.js -lproxyfs.js -lworkerfs.js"])
-    AS_VAR_APPEND([LINKFORSHARED], [" -sEXPORTED_RUNTIME_METHODS=FS,callMain,ENV"])
+    AS_VAR_APPEND([LINKFORSHARED], [" -sEXPORTED_RUNTIME_METHODS=FS,callMain,ENV,HEAPU32"])
```

Comment:
Oops.

Reply:
Can you provide more information?  I made this change since `python.worker.mjs` is currently trying to look in `HEAPU32` to find Python's version info but it wasn't accessible (in my original builds, I just manually edited `python.worker.mjs` to hard-code the version number to make things work).

I'm guessing from "Oops," though, that this isn't the right way to fix this?  First time messing with Emscripten/WASM at all, so if you don't mind, more context would be appreciated.

Reply:
@hoodmane The "oops" comment is ambiguous to me as well - just to confirm, this is an "oops, HEAPU32 should definitely be included, how did I miss that?", rather than or "oops, this shouldn't be included"?